test/run_envoy_fuzz_coverage.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+set -e
+
+[[ -z "${SRCDIR}" ]] && SRCDIR="${PWD}"
+
+echo "Starting run_envoy_fuzz_coverage.sh..."
+echo "    PWD=$(pwd)"
+echo "    SRCDIR=${SRCDIR}"
+
+# This is the fuzz target that will be run to generate coverage data.
+if [[ $# -gt 0 ]]; then
+  FUZZ_TARGETS=$*
+else
+  # By default, this will be all fuzz targets.
+  FUZZER_TARGETS_CC=$(find . -name *_fuzz_test.cc)
+  FUZZER_TARGETS="$(for t in ${FUZZER_TARGETS_CC}; do echo "${t:2:-3}"; done)"
+  FUZZ_TARGETS=""
+  for t in ${FUZZER_TARGETS}
+  do
+    FUZZ_TARGETS+="//"$(dirname "$t")":"$(basename "$t")" "
+  done
+fi
+
+echo ${FUZZ_TARGETS}
+
+# Build all fuzz targets to run instrumented with libfuzzer in sequence.
+echo "Building fuzz targets..."
+for t in ${FUZZ_TARGETS}
+do
+  bazel build "${t}_with_libfuzzer" --config asan-fuzzer -c opt
+done
+
+# Now run each fuzz target in parallel for 60 seconds.
+pids=""
+TEMP_CORPORA=""
+echo "Running fuzz targets..."
+for t in ${FUZZ_TARGETS}
+do
+  # Get the original corpus for the fuzz target
+  ORIGINAL_CORPUS=$(bazel query "labels(srcs, ${t}_corpus)" | head -1)
+  ORIGINAL_CORPUS=${ORIGINAL_CORPUS/://}
+  ORIGINAL_CORPUS=$(dirname ${ORIGINAL_CORPUS})
```

Comment:
Nit: all these `$(some_dir_function ..)` should be `"$(some_dir_function ..)"` for whitespace safety.